views/new_peep.erb

```diff
@@ -0,0 +1,18 @@
+
+<body style='background-color: #87CEFA'>
+
+<div style= 'padding:10px; border-radius: 10px; 
+ color: white; margin: auto; display:grid; font-size: large; 
+ position:absolute; top: 50%; left: 50%; transform: translate(-50%, -50%);'>
+<form style= 'padding:30px'
+    action="/peeps" method="post">
+   <input type="text" name="message" style= 'padding:15px' >
+   <input type="submit" value="Peep!" style= 'padding:8px'/>
+ </form>
```

Comment:
Did this work? I can't actually see where you pass the time through to the controller here...